common/app/model/content.scala

```diff
@@ -267,6 +267,14 @@ class Content protected (val apiContent: ApiContentWithMeta) extends Trail with
     }
   }
 
+// This is a fix for the hanging letters on long section/series labels on the leftCol breakpoint.
+  lazy val areLabelsLong: Boolean = {
+    val sectionLabelList: Seq[String] = sectionLabelName.split(" ").toSeq
+    val blogLabelList: Seq[String] = blogOrSeriesTag.map(_.name.split(" ").toSeq).getOrElse(Nil)
+    val filteredList: Seq[String] = sectionLabelList.filter(_.length > 10) ++ blogLabelList.filter(_.length > 12)
```

Comment:
Blog labels are smaller with a lower font-weight. 

I do think we should be checking individual words as opposed to the length of the entire tag. For instance, we don't want to force a tag of "General election 2015" smaller as it will fit fine at the larger size.